jvm/src/main/scala/frankenpaxos/epaxos/ClientMain.scala

```diff
@@ -47,13 +49,20 @@ object ClientMain extends App {
   val config = ConfigUtil.fromFile(flags.paxosConfigFile.getAbsolutePath())
   val paxosClient =
     new Client[NettyTcpTransport](address, transport, logger, config)
+  val commandsToRun = Source.fromFile("/Users/neil/Documents/Databases/frankenpaxos/zeno/jvm/src/main/scala/frankenpaxos/epaxos/commands.txt").getLines().toList
```

Comment:
Eventually, clients should probably generate commands on their own based on some command line parameters on conflict distribution.